rpcd/playbooks/roles/filebeat/defaults/main.yml

```diff
@@ -154,6 +154,13 @@ filebeat_logging_paths:
       pattern: "{{ multiline_openstack_pattern }}"
       negate: 'true'
       match: after
+  - paths:
```

Comment:
If the answer to those is both "yes", then you should be able to copy-pasta the 

```py
multiline:
  pattern: ...
  negate: 'true'
  match: after
```

Stanza from above